packages/node_modules/@ciscospark/widget-recents/src/components/NoSpaces.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import styles from './NoSpaces.css';
+
+const propTypes = {
+  title: PropTypes.string.isRequired,
+  emptyMessage: PropTypes.string.isRequired
+};
+
+const defaultProps = {
```

Comment:
No need for an empty default props

Reply:
oops, good catch. copy pasta i forgot to delete :)